src/query/functions/temporal/aggregation_test.go

```diff
@@ -223,6 +235,18 @@ var testCasesNaNs = []testCase{
 			{math.NaN(), math.NaN(), math.NaN(), math.NaN(), math.NaN()},
 		},
 	},
+	{
+		name:   "quantile_over_time",
+		opType: QuantileType,
+		afterBlockOne: [][]float64{
+			{math.NaN(), math.NaN(), math.NaN(), math.NaN(), math.NaN()},
```

Comment:
Do we have some tests that don't include `NaN`'s? Might be a useful simple case to cover.

Reply:
Yeah we should add tests for all of the temporal functions with some NaNs - ill do this in a separate PR.